src/Microsoft.VisualStudio.Threading.Shared/AsyncLazy.cs

```diff
@@ -150,7 +150,15 @@ public Task<T> GetValueAsync(CancellationToken cancellationToken)
                         Func<Task<T>> valueFactory = async delegate
                         {
                             await resumableAwaiter;
-                            return await originalValueFactory().ConfigureAwait(continueOnCapturedContext: false);
+                            try
```

Comment:
It shouldn't matter since that awaitable never throws. But I've moved it so it's more obviously correct.